src/index.js

```diff
@@ -6,7 +6,9 @@ import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { ApiProvider } from "./context/apiContext/api";
 import { SingleVideoProvider } from "./context/singleVideoContext/singleVideoContext";
-;
+import { LikedVideoProvider } from "./context/likedVideoContext/likedVideoContext";
+import { DisLikedVideoProvider } from "./context/disLikedVideoContext/disLikedVideoContext";
+
```

Comment:
same here, export all contexts from the index.js file in the context folder and import contexts from the index.js file present in the context folder.